Jenkinsfile

```diff
@@ -19,19 +19,13 @@ pipeline {
 				}
 			}
 			steps {
-				configFileProvider([
-					configFile(fileId: 'a79c37a7-bfd9-4699-8077-f992027b2c1b', targetLocation: '/home/jenkins/.m2/settings.xml'),
-					configFile(fileId: 'b78b4e15-215f-42bc-81ac-53cd3e2ef708', targetLocation: '/home/jenkins/.m2/')
```

Comment:
Don't we need the new GPG keyring import described by Frederic?

Reply:
> Adding GPG config would provide new functionality to the build. This is not migration issue. Build will work without GPG, keeping already existed functionality.

@senivam If you check the `pom.xml` file https://github.com/eclipse-ee4j/jaxrs-api/blob/61a230d5924e00714fb0799a4c16175a74c0dd29/jaxrs-api/pom.xml#L125 you will see that `maven-gpg-plugin` was used *already before the migration* to `sign` the JARs. So this is broken *by the migration*, hence should get fixed *by the same*  "fix-the-miration-PR".

Reply:
yes, maven-gpg-plugin was used before, it is correct. However those to lines which are currently breaking the build are substitution for settings.xml and settings-security.xml which were provided inside job's configuration in previous JIPP. The current JIPP has mounted those files (settings.xml and settings-security.xml) just before the build (you can check it in log of failed builds after migration). So, there is no need to provide them inside job's configuration. Further more, there is no references to those configuration files in the actual configuration. That is why job keeps failing (this is visible in log output as well). There is no mention about GPG sign failure or sign padding failure (which would be sign of wrong GPG functioning). So, when those two lines will be removed from job's configuration, job will work.

Reply:
btw in pom.xml gpg plugin is being activated when used with -Prelease profile. The job we are talking about performs nightly builds only so no profile release (thus no option -Prelease) is used when running the job.

Reply:
I do not agree that we are talking only about one particular job (nightly builds), as the word "nightly" was *first* mentioned by your comment directly above. Your PR is called "Migration of Jenkins files" (which includes the GPG keyring), and your initial comment says: "It's not required to use configuration files for Jenkins jobs after migration to ci.eclipse.org". If you like a second PR for the GPG fix, that's fine for me, but we should clearly rephrase *this* PR's title and description so reviewers do know that all you intend is to fix solely the *nightly* build.

Reply:
where else is this Jenkins file used?

Reply:
which contains stage:
`stage('Nightly Build') {`

Reply:
It needed some time to think about your last two comments, but now I see what you mean. GPG is not used by the *pipeline jobs* (hence `Jenkinsfile`) plays no role, while the jobs using GPG already perform the keyring import using their particular *static* configuration in Jenkins.